packages/editor/src/components/post-publish-panel/index.js

```diff
@@ -61,22 +61,21 @@ class PostPublishPanel extends Component {
 		this.setState( { loading: true } );
 	}
 
+	onUpdateDisableToggle() {
+		// There should be a setting to allow users to skip the pre-publish step forever.
```

Comment:
That'll need to be a user setting, I'd think. I don't know how we set those in Gutenberg, but this should dispatch an action that makes an API request to toggle said setting.